ynput/OpenPype
# Houdini: Introduce imprint function for correct version in hda loader

## Changelog Description
Resolve #5478 

## Additional info
n/a

## Testing notes:
1. Launch Houdini via Launcher
2. Load Hda
3. Manage
4. Set version or update to the latest
5. Should be at the right version
